torch/jit/supported_ops.py

```diff
@@ -230,7 +230,7 @@ def _get_global_builtins():
 
     magic_methods_rows = []
     for fn, magic_method in magic_methods:
-        magic_methods_rows.append('":any:`{}`", "``{}``"'.format(fn, magic_method))
+        magic_methods_rows.append('"{}", "``{}``"'.format(fn, magic_method))
```

Comment:
the linking here did not work so I removed it.

Reply:
We should probably inform whoever wrote this in the first place

Reply:
FYI @suo

Reply:
That would be @driazati in gh-30347

Reply:
They're not that important, they were just linking to the Python docs for built in functions like `int()` and `str()`